server/src/main/java/io/deephaven/server/appmode/ApplicationInjector.java

```diff
@@ -89,4 +88,14 @@ private void loadApplication(final Path applicationDir, final ApplicationConfig
             ticketResolver.onApplicationLoad(app);
         }
     }
+
+    private void loadApplicationFactory(ApplicationState.Factory factory) {
+        log.info().append("Starting ApplicationState.Factory '").append(factory.toString()).append('\'').endl();
+        try (final SafeCloseable ignored = LivenessScopeStack.open()) {
```

Comment:
We probably have to supply an ExecutionContext before using the factory to create the application. The most straightforward solution is likely going to be opening the script session's EC provided via: 

```java
try (final SafeCloseable ignored1 = LivenessScopeStack.open();
    final SafeCloseable ignored2 = scriptSessionProvider.get().getExecutionContext().open()) {
        // ...
}
```